onnxruntime/python/tools/kernel_explorer/kernels/gemm_test.py

```diff
@@ -10,20 +10,11 @@
 import pytest
 
 
-def dtype_to_bytes(dtype):
-    type_map = {
-        "float16": 2,
-        "float32": 4,
-    }
-    return type_map[dtype]
-
-
-def dtype_to_funcs(dtype):
-    type_map = {
-        "float16": list(filter(lambda x: "RocblasGemm_half" in x, dir(ke))),
-        "float32": list(filter(lambda x: "RocblasGemm_float" in x, dir(ke))),
-    }
-    return type_map[dtype]
+def dtype_to_suffix(dtype):
+    return {
+        "float32": "float",
+        "float16": "half",
+    }[dtype]
 
 
 def _test_gemm(func, dtype: str, m: int, n: int, k: int, transa=False, transb=False):
```

Comment:
Looks like there are lots of test failures?

Reply:
All of them should be accumulated round-off error. We need a better `atol` and `rtol` from backward error analysis to avoid them. I think it will/might happen when the accumulation axis (k-axis) is large enough.

Reply:
Can we increase atol/rtol, or reduce k dimensions?

(Having lots of false negatives makes it a bit harder to expose true negatives)

Reply:
They should be a function of k, will figure it out to make it more robust.

Reply:
There should be no false positive now.